### PR TITLE
Allow to use peds and vehicles as a camera target

### PR DESCRIPTION
**Pull request by TheNormalnij**
Allow to use peds and vehicles as a camera target in setCameraTarget function.
- [x] Clientside
- [x] Serverside

### CODE REVIEW

File: Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp
Comment (Citizen01) [inline Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp]: Useless comment now.

File: Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp
Comment (Citizen01) [inline Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp]: No, don't brake it and keep it to `false` there.

**Review by Citizen01 — CHANGES_REQUESTED** [review]
Thanks for the PR. However, if it was this easy to add support for new element types as camera target, what blocks us to support all entity types ?
(I think there might be a valid reason for it to only support players in the 1st place but I don't know what it is then)

**Comment by TheNormalnij** [comment]
I tried with objects but it didn't work.
Now i have bug with getCameraTarget. It need to be fixed.

**Comment by TheNormalnij** [comment]
setCameraViewMode works only with 1-4 modes. Maybe it can be fixed later.

**Review by sbx320 — APPROVED** [review]
We'd need to lock this behaviour behind a min MTA version check, as otherwise this will cause issues with new vs old clients. Aside from that it looks good.  

So **before merging** (whoever does it): Please add a version check with an applicable version

**Comment by TheNormalnij** [comment]
@sbx320 do you mean MinClientReqCheck and MinServerReqCheck or pBitStream->Version?

**Review by Citizen01 — APPROVED** [review]

**Comment by Citizen01** [comment]
Requires a min MTA version check still.
Do not merge yet.

**Comment by TheNormalnij** [comment]
Added

File: Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp
Comment (qaisjp) [inline Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp:4506]: Wrong bitstream you need to find the latest one

**Pull request merged**
